src/sidebar/test/annotation-ui-test.js

```diff
@@ -483,28 +491,12 @@ describe('annotationUI', function () {
   });
 
   describe('#updatingAnchorStatus', function () {
-    it("updates the annotation's tag", function () {
-      var annot = defaultAnnotation();
-      annotationUI.addAnnotations([annot]);
-      annotationUI.updateAnchorStatus(annot.id, 'atag', true);
-      assert.equal(annotationUI.getState().annotations[0].$tag, 'atag');
-    });
-
     it("updates the annotation's orphan flag", function () {
       var annot = defaultAnnotation();
       annotationUI.addAnnotations([annot]);
-      annotationUI.updateAnchorStatus(annot.id, 'atag', true);
+      annotationUI.updateAnchorStatus({ [tagForID(annot.id)]: 'orphan' });
       assert.equal(annotationUI.getState().annotations[0].$orphan, true);
     });
-
-    it('updates annotations by tag', function () {
```

Comment:
This test is now irrelevant since there is only one way to reference which annotation was anchored / failed to anchor, and that is covered by the test above.